libs/community/langchain_community/chat_models/reka.py

```diff
@@ -152,8 +154,9 @@ class RekaCommon(BaseLanguageModel):
     default_request_timeout: Optional[float] = None
     max_retries: int = 2
     reka_api_key: Optional[str] = None
-    count_tokens: Optional[Callable[[str], int]] = None
     model_kwargs: Dict[str, Any] = Field(default_factory=dict)
+    model_config = ConfigDict(extra="forbid")
+    _tiktoken_encoder = None
```

Comment:
why is this private? could we make it into a more general `token_counter` param with better types? something like this https://github.com/langchain-ai/langchain/blob/master/libs/core/langchain_core/messages/utils.py#L577